LDM-148.tex

```diff
@@ -922,12 +930,17 @@ \subsection{Interfaces}\label{backbone-interfaces}
 
 \section{Software Components}\label{software-components}
 
+DM is responsible for delivering a set of operational services, as described above, and most of those are implemented by custom or customized software components.
+The implementing software products are distinct from the service products as they do not include configurations needed for deploying the service in different environments.
+This section describes some of the software products that are not directly tied to corresponding services.
+
 \subsection{Science Payloads}\label{science-payloads}
 
 These payloads are described in more detail in the DM Applications Design Document \citedsp{LDM-151}.
-Payloads are built from application software components.
+Payloads are built from application software components from the Science Pipelines Libraries or third-party products.
+They specify inputs, outputs, and configurations as well as the order of execution of science algorithms from those libraries.
 Most payloads execute under control of the Batch Production service.
-Exceptions include the Alert Production Payload, the Raw Calibration Validation Payload, and the Daily Calibration Products Update Payload.
+Exceptions include the Alert Production Payload and some of the Calibration Software.
```

Comment:
“...and some Calibration Payloads”?

Reply:
Because Calibration is a mixture of automated and potentially manual code executing in a variety of environments (likely including Kafka/InfluxDB), naming it as "Payloads" seemed inappropriate.